Flutter/pdf-viewer/How-to/Add-SignaturePad-signatures-to-Syncfusion-Flutter-PdfViewer.md

```diff
@@ -1,17 +1,17 @@
 ---
 layout: post
 title: Add digital signature in SfPdfViewer using SfSignaturePad | Syncfusion
-description: Learn here all about how to add SfSignaturePad signatures in the Syncfusion Flutter PDF Viewer (SfPdfViewer) widget and more.
+description: Learn here all about how to add SfSignaturePad signatures in the Syncfusion® Flutter PDF Viewer (SfPdfViewer) widget and more.
 platform: flutter
```

Comment:
What is the reason for handling two variations here?

<img width="970" alt="image" src="https://github.com/user-attachments/assets/347f8527-9848-49f7-b27f-e5876f5b9fff" />

Reply:
In CI validation, the description needs to have only 100 and 160 characters. When using <sup>&reg;</sup>, the character length increases.
For the above reason and the description won't be visible in the live documentation site, and to follow similar structure in the description, I used only the `®` character in the description.

Reply:
Ok. Please mention this note in description also. So that the reviewers will understand in the code.

Reply:
I have added this note in the PR description.